src/pages/MsigUI.vue

```diff
@@ -2,40 +2,43 @@
 	<a-layout-content class="content_sign">
 		<a-row>
 			<a-col :xs="{ span: 24 }" :lg="{ span: 10 }">
-				<h1>Unsigned transaction</h1>
+				<h3>Partial sign transaction</h3>
 				<div class="sign_field">
 					<a-textarea
 						v-model:value="unsignedJson"
-						placeholder="Base64 or JSON object"
+						placeholder="Base64 msgpack"
 						:rows="20"
 						allow-clear
-					/>
+					>
+					</a-textarea>
 				</div>
-				<span class="switch_button">
-					<a-switch
-						v-model:checked="inputBase64"
-						checked-children="MSGPACK"
-						un-checked-children="JSON"
-					/>
-				</span>
 				<a-button type="primary" @click="sign">SIGN</a-button>
+				<MultisigParameters :inputBase64="unsignedJson" />
 			</a-col>
 			<a-col :xs="{ span: 24 }" :lg="{ span: 12, offset: 2 }">
-				<h1>Signed transaction</h1>
+				<h3>Signed transaction</h3>
 				<a-card
-					style="width: 100%"
+					class="card"
 					title="Format Transaction"
 					:tab-list="tabList"
 					:active-tab-key="key"
+					:bordered="false"
 					@tabChange="(key) => onTabChange(key, 'key')"
 				>
 					<template #customTab="item">
 						<span>
 							{{ item.key }}
 						</span>
 					</template>
-					{{ contentList[key] }}
+					<a-textarea
+						style="background-color: white !important; color: black !important"
```

Comment:
styles to be added using classes, I think similar is there is txSender, update there as well